hack/clone-api-files.sh

```diff
@@ -81,6 +76,14 @@ function remove_outdated_api_files() {
     done
 }
 
+echo ">> remove excluded files from ${KOORDINATOR_REPO_DIR}"
+for excluded_path in "${EXCLUDED_PATHS[@]}";
+do
+    rm_path="${KOORDINATOR_REPO_DIR}/${excluded_path}"
+    rm -rf ${rm_path}
```

Comment:
for safety, need to verify if the directory is valid and under the repo directory

Reply:
it is safety since KOORDINATOR_REPO_DIR is under tmp dir like "/tmp.0V7kXPcRQv".
unless setting "EXCLUDED_PATHS" with "../../xxx/" by purpose.